core/src/main/java/org/apache/iceberg/util/SerializationUtil.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.iceberg.mr;
+package org.apache.iceberg.util;
```

Comment:
I would rather not expose this class in core just for tests. Could you copy the methods into `TestHelpers` or embed them in the test suites instead? That would cut down on a lot of files touched by this PR.

Reply:
Reverted this change